src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2401,6 +2401,30 @@ def get_disable_cost_analysis(self) -> bool:
         # because it's already checked in _get_enable_cost_analysis
         return self.raw_param.get("disable_cost_analysis")
 
+    def get_ai_toolchain_operator(self) -> Union[bool, None]:
+        """Obtain the value of ai_toolchain_operator.
+         When enabled, if both enable_ai_toolchain_operator and disable_ai_toolchain_operator are
+        specified, raise a MutuallyExclusiveArgumentError.
+        :return: bool or None
+        """
+        # this parameter does not need dynamic completion
+        # validation is done with param validator
+        enable_ai_toolchain = self.raw_param.get("enable_ai_toolchain_operator")
+        disable_ai_toolchain = self.raw_param.get("disable_ai_toolchain_operator")
+
+        if enable_ai_toolchain is False and disable_ai_toolchain is False:
+            return None
+
+        if enable_ai_toolchain and disable_ai_toolchain:
+            raise MutuallyExclusiveArgumentError(
+                "Cannot specify --enable-ai-toolchain-operator and "
+                "--disable-ai-toolchain-operator at the same time. "
+            )
+        if enable_ai_toolchain is not None:
+            return enable_ai_toolchain
+        if disable_ai_toolchain is not None:
+            return not disable_ai_toolchain
```

Comment:
will also return `True` even if you want to disable the feature

Reply:
Addressed